src/test/java/net/sf/jabref/logic/util/strings/StringUtilTest.java

```diff
@@ -313,6 +312,9 @@ public void testExpandAuthorInitialsDoNotAddDot() {
         assertEquals("Moore, O.", StringUtil.expandAuthorInitials("Moore, O."));
         assertEquals("Moore, A. O.", StringUtil.expandAuthorInitials("Moore, A. O."));
         assertEquals("Moore, A.-O.", StringUtil.expandAuthorInitials("Moore, A.-O."));
+        assertEquals("MEmre", StringUtil.expandAuthorInitials("MEmre"));
+        assertEquals("{\\'{E}}douard", StringUtil.expandAuthorInitials("{\\'{E}}douard"));
```

Comment:
No it will not. The dot will be added only if the uppercase letter is followed by a whitespace, dash or tilde. I think it would be to hard to detect all the latex stuff.